QualificationsFramework/Examples/ACQF.json

```diff
@@ -0,0 +1,184 @@
+{
+	"@graph": [
+		{
+			"@id": "res:acqf",
+			"@type": "???:QualificationsFramework",
+			"ceasn:name": { "en": "African Continental Qualifications Framework" },
+			"ceterms:alternateName": { "en": "ACQF" },
+			"ceasn:description": { "en": "The ACQF is a comprehensive and inclusive meta-referencing qualifications framework, designed to support a holistic and systemic vision of learning, qualifications, and credentials. The ACQF is oriented to lifelong learning and supports parity of esteem of learning outcomes acquired in formal, non-formal and informal contexts, and all levels of education and training." },
+			"ceasn:dateValidFrom": "2022-11-01",
+			"ceterms:versionCode": "17",
+			"ceasn:source": [
+				"https://drive.google.com/file/d/1BnU5VOqS6hu2YRhowC7x6J1R1_p6UblV/view"
+			],
+			"ceasn:hasGlossary": [
+				"res:acqfGlossary"
+			],
+			"asn:hasProgressionModel": [
+				"res:acqfLevels"
+			]
+		},
+		{
+			"@id": "res:acqfGlossary",
+			"@type": "skos:ConceptScheme",
+			"ceasn:name": { "en": "ACQF Acronym Glossary" },
+			"skos:hasTopConcept": [
+				"res:acqfGlossary_acqf",
+				"res:acqfGlossary_acts",
+				"etc"
+			]
+		},
```

Comment:
for info: as well as the abbreviations used at the front of the doc, there is a Glossary in Annex 1. It is the latter that I had in mind for the Glossary concept scheme. This is valid too. Does anyone think we need to make a distinction between "abbreviations used" and "glossary" in our vocabulary?

Reply:
I added another scheme for that glossary and updated the JSON for the abbreviations.

I don't think a distinction is really necessary, and I'm sure there would be cases where abbreviations and definitions get intermingled in the same scheme.